lib/plugins/Label_10_Slash.ts

```diff
@@ -0,0 +1,87 @@
+import { DateTimeUtils } from '../DateTimeUtils';
+import { DecoderPlugin } from '../DecoderPlugin';
+import { DecodeResult, Message, Options } from '../DecoderPluginInterface';
+import { Waypoint } from '../types/waypoint';
+import { CoordinateUtils } from '../utils/coordinate_utils';
+import { ResultFormatter } from '../utils/result_formatter';
+import { RouteUtils } from '../utils/route_utils';
+
+export class Label_10_Slash extends DecoderPlugin { // eslint-disable-line camelcase
+  name = 'label-10-slash';
+
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ['10'],
+      preambles: ['/'],
+    };
+  }
+
+  decode(message: Message, options: Options = {}): DecodeResult {
+    const decodeResult = this.defaultResult();
+    decodeResult.decoder.name = this.name;
+    decodeResult.formatted.description = 'Position Report';
+    decodeResult.message = message;
+
+    const parts = message.text.split('/');
+    if (parts.length < 17) {
+      if (options?.debug) {
+        console.log(`Decoder: Unknown 10 message: ${message.text}`);
+      }
+      decodeResult.remaining.text = message.text;
+      decodeResult.decoded = false;
+      decodeResult.decoder.decodeLevel = 'none';
+      return decodeResult;
+    }
+
+    const lat = parts[1];
+    const lon = parts[2];
+    const position = {
+      latitude: (lat[0] === 'N' ? 1 : -1) * Number(lat.substring(1)),
```

Comment:
I think we could start extracting this kind of stuff out to the utility functions perhaps?

Reply:
yes but... all of these messages are slightly different i run into "naming things" issues... totally open to suggestions. I would put it in `CoordinateUtils`